models/staging/src_reddit_ads.yml

```diff
@@ -151,12 +145,6 @@ sources:
             description: '{{ doc("ad_id") }}'
           - name: clicks
             description: '{{ doc("clicks") }}'
-          - name: comment_downvotes
-            description: '{{ doc("comment_downvotes") }}'
-          - name: comment_upvotes
-            description: '{{ doc("comment_upvotes") }}'
-          - name: comments_page_views
-            description: '{{ doc("comments_page_views") }}'
```

Comment:
These columns (and same in the other lines for these cols) only existed in the source and were never brought into staging.